builder/dockerfile/metrics.go

```diff
@@ -0,0 +1,29 @@
+package dockerfile
+
+import "github.com/docker/go-metrics"
+
+var (
+	buildsTriggered metrics.Counter
+	buildsFailed    metrics.LabeledCounter
+)
+
+func init() {
```

Comment:
Great suggestion @cpuguy83! I should have done that in the first place to avoid the issues you mentioned. I've created a package called `internal` with `builder/internal/constants.go` file in it. WDYT?